Tests/XCUITests/PhotonActionSheetTests.swift

```diff
@@ -64,11 +64,14 @@ class PhotonActionSheetTests: BaseTestCase {
 
         mozWaitForElementToExist(app.buttons[AccessibilityIdentifiers.Toolbar.shareButton], timeout: 10)
         app.buttons[AccessibilityIdentifiers.Toolbar.shareButton].tap()
+        mozWaitForElementToExist(app.navigationBars["UIActivityContentView"], timeout: TIMEOUT)
+        mozWaitForElementToExist(app.collectionViews.cells["Copy"])
+        mozWaitForElementToExist(app.collectionViews.cells["Send Link to Device"])
+        mozWaitForElementToExist(app.collectionViews.cells["Markup"])
+        mozWaitForElementToExist(app.collectionViews.cells["Print"])
```

Comment:
Sometimes, the `fennecElement` is tapped without it is ready. 😞

Reply:
I would pick just one element to wait for not all of them... otherwise this will add more flakiness if one of those is not there, I have seen that ui changing very frequently...